pkg/integration/cedar_test.go

```diff
@@ -0,0 +1,38 @@
+package integration
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+
+	"github.com/cmsgov/easi-app/pkg/cedar"
+	"github.com/cmsgov/easi-app/pkg/handlers"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// Since we can't always hit the CEDAR API in tests
+// (both for performance and VPN reasons)
+// this test can be run in local environments to make sure CEDAR
+// is set up.
+// Other tests should mock the API
+func (s *IntegrationTestSuite) TestCEDARConnection() {
+	req, err := http.NewRequest("GET", "/systems/", nil)
+	s.NoError(err)
+	rr := httptest.NewRecorder()
+
+	cedarClient := cedar.NewTranslatedClient(os.Getenv("CEDAR_API_KEY"))
```

Comment:
Can't access the viper config from here since we're shelling out in `cmd` to run tests. Maybe we just want to stop doing that? It seems like the right move when I set it up, but might be more trouble than worth.

Reply:
We're running the tests from here: https://github.com/CMSgov/easi-app/blob/master/cmd/easi/test/test.go#L21. Since it's shelling out to `go test`, we lose the ability to use viper to pull in configs. I'm suggesting we either fix how we're calling tests, or possibly just run `go test` directly.